Dockerfile

```diff
@@ -217,4 +217,6 @@ COPY --from=git-prod --chown=www-data:www-data /opt/kimai /opt/kimai
 RUN export COMPOSER_HOME=/composer && \
     composer install --working-dir=/opt/kimai --no-dev --optimize-autoloader && \
     composer clearcache
-ENV APP_ENV=prod
+
+FROM ${VER}
```

Comment:
@tobybatch  why added this?

Reply:
When the file built it was always the prod image.  Was I doing something wrong?  This let me switch between dev and prod builds via var.

Look at the build.sh in ./bin

If there is a better way, I'll change it.

Reply:
havent read my answer in slack?
```
you can use the --target=STAGE_NAME flag to build a specific stage...
default is the last stage, so every build is the production. if you want to make an dev build add a --target=dev (bearbeitet) 

see https://docs.docker.com/engine/reference/commandline/build/#specifying-target-build-stage---target
```